packages/rollup/test/multi_linker/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "generated_file_test")
+load("//packages/rollup:index.bzl", "rollup_bundle")
+load("//packages/typescript:index.bzl", "ts_project")
+
+ts_project(
+    name = "tsconfig",
+    srcs = glob(["*.ts"]),
+    deps = ["@rollup_test_multi_linker_deps//@types"],
+)
+
+# Intentionally test with only generated files by transpiling with ts_project
+# to test multi-linker build action with no source inputs.
+# NB: rollup.config.js is used to generated "_{name}.rollup_config.js" in output
+# tree. Only the generated config is a input to the rollup build action.
+rollup_bundle(
```

Comment:
We don't need to enable workers to reproduce it?

Reply:
no this one is just a missing link in the subdir. I don't think we have much test coverage for rollup workers? so that might need some separate verification

Reply:
Oh, for some reason I was assuming this was a fix for https://github.com/bazelbuild/rules_nodejs/issues/2437 which I think is only for workers.

Could always run the test twice with + without workers though 🤷

Reply:
#2437 is an unrelated issue. Hoping to look at that over the weekend. This bug was specific to multi-linker and would be same failure mode with workers or no workers.